.github/workflows/windows.yml

```diff
@@ -22,6 +22,10 @@ env:
   CMAKE_CXX_COMPILER_LAUNCHER: ccache
   CMAKE_C_COMPILER_LAUNCHER: ccache
   CCACHE_MAXSIZE: 500Mi
+  ARTIFACTS_SHARE: '/mount/build-artifacts'
```

Comment:
is it correct windows path?

Reply:
store_artifacts is running on Linux host